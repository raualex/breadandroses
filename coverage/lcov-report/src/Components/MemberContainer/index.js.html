<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/Components/MemberContainer/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/Components/MemberContainer</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>4/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">50% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>6/12</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>4/4</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import cardImages from '../../Utils/cardImages'
import './MemberContainer.css'
import uuid from 'uuid';
&nbsp;
const MemberContainer = (props) =&gt; {
&nbsp;
  const members = props.congress.map((person) =&gt; {
    
    return (
      &lt;div key={uuid()} className="member"&gt;
        &lt;img src={cardImages[person.name]} alt={person.name}/&gt;
        &lt;ul className="member-list"&gt;
          &lt;li&gt;Name:&lt;/li&gt;
          &lt;li&gt;{person.name}&lt;/li&gt;
          &lt;li&gt;Party:&lt;/li&gt;
          &lt;li&gt;{person.party}&lt;/li&gt;
          &lt;li&gt;State:&lt;/li&gt;
          &lt;li&gt;{person.state}&lt;/li&gt;
          &lt;li&gt;Official Website:&lt;/li&gt;
          &lt;li&gt;{person.website}&lt;/li&gt;
          &lt;li className={(person.twitter ? 'show' : <span class="branch-1 cbranch-no" title="branch not covered" >'hidden')</span>}&gt;Twitter:&lt;/li&gt;
          &lt;li className={(person.twitter ? 'show' : <span class="branch-1 cbranch-no" title="branch not covered" >'hidden')</span>}&gt;{person.twitter}&lt;/li&gt;
          &lt;li className={(person.facebook ? 'show' : <span class="branch-1 cbranch-no" title="branch not covered" >'hidden')</span>}&gt;Facebook:&lt;/li&gt;
          &lt;li className={(person.facebook ? 'show' : <span class="branch-1 cbranch-no" title="branch not covered" >'hidden')</span>}&gt;{person.facebook}&lt;/li&gt;
          &lt;li className={(person.phone_number ? 'show' : <span class="branch-1 cbranch-no" title="branch not covered" >'hidden')</span>}&gt;Phone:&lt;/li&gt;
          &lt;li className={(person.phone_number ? 'show' : <span class="branch-1 cbranch-no" title="branch not covered" >'hidden')</span>}&gt;{person.phone_number}&lt;/li&gt;
        &lt;/ul&gt;
      &lt;/div&gt;
    )
  })
&nbsp;
  return (
    &lt;div className="members-container"&gt;
      { members }
    &lt;/div&gt;
  )
&nbsp;
}
&nbsp;
export default MemberContainer;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sat Nov 03 2018 16:05:56 GMT-0600 (MDT)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
