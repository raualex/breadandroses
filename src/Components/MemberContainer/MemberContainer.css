.committee-title {
  -webkit-text-stroke: 2px #000;
  color: #f4c076;
  font-family: 'Black Han Sans', sans-serif;
  font-size: 2.5em;
}

.hidden {
  display: none;
}

.R,
.D,
.I {
  background-color: rgba(243,243,243,0.8);
  border-radius: 48px;
  flex: 0 0 auto;
  height: 676px;
  margin: 8px;
  padding: 28px;
  width: 294px;
}

.D {
  border: solid 6px blue;
}

.I {
  border: solid 6px yellow;
}

.R {
  border: solid 6px red;
}

.label {
  font-weight: 700;
}

.loading-container {
  align-items: center;
  display: flex;
  justify-content: center;
  width: 100%;
}

.members-container {
  display: flex;
  height: 760px;
  flex-wrap: nowrap;
  overflow-x: auto;
}

.member-details {
  align-items: center;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
}

.member-list {
  list-style-type: none;
  padding: 0;
}

.politician {
  border-radius: 48px; 
}

.state-filter-container {
  align-items: center;
  display: flex;
  height: 100px;
  justify-content: center;
  width: 100%;
}

.state-select {
  font-size: 1.5em;
  height: 60px;
  margin-right: 80px;
  width: 500px;
}

.reset-btn {
  background-color: #f4c076;
  border: solid 4px #000;
  font-family: 'Black Han Sans', sans-serif;
  font-size: 1.5em;
  height: 60px;
  width: 300px;
}

.reset-btn:active {
  background-color: #CC2B1D;
  color: #fff;
}

.reset-btn:focus {
  outline: none;
}

.reset-btn:hover {
  cursor: pointer;
}
