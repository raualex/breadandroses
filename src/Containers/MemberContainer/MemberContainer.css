.hidden {
  display: none;
}

.member {
  background-color: rgba(243,243,243,0.8);
  border-radius: 48px;
  flex: 0 0 auto;
  margin: 8px;
  padding: 4px;
}

.members-container {
  display: flex;
  height: 500px;
  flex-wrap: nowrap;
  overflow-x: auto;
}

.member-list {
  list-style-type: none;
  padding: 0;
}
