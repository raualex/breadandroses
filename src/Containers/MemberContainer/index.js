import React from 'react';
import { connect } from 'react-redux';
import uuid from 'uuid';

const MemberContainer = (props) => {

  const members = props.senate.map((senator) => {
    return (
      <div key={uuid()} className="senateMember">
        <ul>
          <li>{senator.name}</li>
          <li>{senator.party}</li>
          <li>{senator.state}</li>
        </ul>
      </div>
    )
  })

  return (
    <div>
      { members }
    </div>
  )

}

const mapStateToProps = (state) => ({
  senate: state.senateMembers
})

export default connect(mapStateToProps)(MemberContainer);