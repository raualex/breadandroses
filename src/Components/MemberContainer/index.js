import React, { Component } from 'react';
import cardImages from '../../Utils/cardImages';
import { senateStates, houseStates } from '../../Utils/stateList';
import Loading from '../Loading';
import './MemberContainer.css';
import PropTypes from 'prop-types';
import uuid from 'uuid';

class MemberContainer extends Component {
  constructor(props) {
    super();

    this.state = {
      select: ''
    }
  }

  createStateList = () => {
    let stateList;
    const { navClicked } = this.props

    if (navClicked === 'senate') {
      stateList = senateStates
    } else if (navClicked === 'house') {
      stateList = houseStates
    }


    return stateList.map(state => {
      let stateName = Object.values(state)
      let stateKey = Object.keys(state)
      return (
        <option key={stateKey} value={stateKey}>
          {stateName}
        </option>
      )
    })
  }

  handleChange = (event) => {
    let state = event.target.value
   
    if (this.state.select === '') {
      this.setState({ select: state }, () => this.props.filterState(state, this.props.navClicked))
    } else {
      return
    }
  }

  handleReset = (event) => {
    event.preventDefault()
    this.setState({ select: '' }, () => this.props.resetFilter())
  }

  render() {
    const { navClicked, congress } = this.props
    let title;
    let members;

    if (navClicked === 'senate') {
      title = <h1 className="committee-title">Committee on Health, Education, Labor, and Pensions</h1>
    } else if (navClicked === 'house') {
      title = <h1 className="committee-title">Committee on Education and the Workforce</h1>
    }

    if (!congress.length) {
      members =
        <div className="loading-container">
          <Loading />
        </div>
    } else {    
      members = congress.map((person) => {
 
        return (
          <div key={uuid()} className={person.party}>
            <img 
              src={cardImages[person.name]} 
              alt={person.name} 
              className="politician"
            />
            <div className="member-details">
              <ul className="member-list">
                <li className="label">Name:</li>
                <li>{person.name}</li>
                <li className="label">Party:</li>
                <li>{person.party}</li>
                <li className="label">State:</li>
                <li>{person.state}</li>
                <li className="label">Official Website:</li>
                <li><a href={person.website} target="_blank" rel="noopener noreferrer">{person.website}</a></li>
                <li className={(person.twitter ? 'label' : 'hidden')}>Twitter:</li>
                <li className={(person.twitter ? 'show' : 'hidden')}>{person.twitter}</li>
                <li className={(person.facebook ? 'label' : 'hidden')}>Facebook:</li>
                <li className={(person.facebook ? 'show' : 'hidden')}>{person.facebook}</li>
                <li className={(person.phone_number ? 'label' : 'hidden')}>Phone:</li>
                <li className={(person.phone_number ? 'show' : 'hidden')}>{person.phone_number}</li>
              </ul>
            </div>
          </div>
        )
      })
    }

    return (
      <div>
        { title }
        <div className="members-container">
          { members }
        </div>
        <form 
          className="state-filter-container">
          <select 
            className="state-select"
            onChange={this.handleChange}
            value={this.state.select}
          >
            <option 
              key="start" 
              value=""
            >--Select your state--
            </option>
            { this.createStateList() }
          </select>
          <button
            className="reset-btn"
            onClick={this.handleReset}
          >Reset State Filter</button>
        </form>
      </div>
    )
  }
}

MemberContainer.propTypes = {
  congress: PropTypes.array,
  navClicked: PropTypes.string,
  filterState: PropTypes.func,
  resetFilter: PropTypes.func
}

export default MemberContainer;