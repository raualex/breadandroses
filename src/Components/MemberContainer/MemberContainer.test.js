import React from 'react';
import MemberContainer from './';
import { shallow } from 'enzyme';
jest.mock('uuid', () => jest.fn(() => 1));

describe('MemberContainer', () => {
  let wrapper;
  let mockCongress = [ { name: 'Bernie Sanders', party: 'I', state: 'VT', website: 'www.sanders.org', twitter: 'sanders', facebook: 'sanders', phone_number: '555-5555' } ]

  beforeEach(() => {
    wrapper = shallow(<MemberContainer congress={mockCongress} />)
  })
  
  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
})
