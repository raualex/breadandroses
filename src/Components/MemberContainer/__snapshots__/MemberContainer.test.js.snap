// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberContainer matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MemberContainer
    congress={
      Array [
        Object {
          "facebook": "sanders",
          "name": "Bernie Sanders",
          "party": "I",
          "phone_number": "555-5555",
          "state": "VT",
          "twitter": "sanders",
          "website": "www.sanders.org",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="member"
        >
          <img
            alt="Bernie Sanders"
          />
          <ul
            className="member-list"
          >
            <li>
              Name:
            </li>
            <li>
              Bernie Sanders
            </li>
            <li>
              Party:
            </li>
            <li>
              I
            </li>
            <li>
              State:
            </li>
            <li>
              VT
            </li>
            <li>
              Official Website:
            </li>
            <li>
              www.sanders.org
            </li>
            <li
              className="show"
            >
              Twitter:
            </li>
            <li
              className="show"
            >
              sanders
            </li>
            <li
              className="show"
            >
              Facebook:
            </li>
            <li
              className="show"
            >
              sanders
            </li>
            <li
              className="show"
            >
              Phone:
            </li>
            <li
              className="show"
            >
              555-5555
            </li>
          </ul>
        </div>,
      ],
      "className": "members-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="Bernie Sanders"
            />,
            <ul
              className="member-list"
            >
              <li>
                Name:
              </li>
              <li>
                Bernie Sanders
              </li>
              <li>
                Party:
              </li>
              <li>
                I
              </li>
              <li>
                State:
              </li>
              <li>
                VT
              </li>
              <li>
                Official Website:
              </li>
              <li>
                www.sanders.org
              </li>
              <li
                className="show"
              >
                Twitter:
              </li>
              <li
                className="show"
              >
                sanders
              </li>
              <li
                className="show"
              >
                Facebook:
              </li>
              <li
                className="show"
              >
                sanders
              </li>
              <li
                className="show"
              >
                Phone:
              </li>
              <li
                className="show"
              >
                555-5555
              </li>
            </ul>,
          ],
          "className": "member",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Bernie Sanders",
              "src": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  Name:
                </li>,
                <li>
                  Bernie Sanders
                </li>,
                <li>
                  Party:
                </li>,
                <li>
                  I
                </li>,
                <li>
                  State:
                </li>,
                <li>
                  VT
                </li>,
                <li>
                  Official Website:
                </li>,
                <li>
                  www.sanders.org
                </li>,
                <li
                  className="show"
                >
                  Twitter:
                </li>,
                <li
                  className="show"
                >
                  sanders
                </li>,
                <li
                  className="show"
                >
                  Facebook:
                </li>,
                <li
                  className="show"
                >
                  sanders
                </li>,
                <li
                  className="show"
                >
                  Phone:
                </li>,
                <li
                  className="show"
                >
                  555-5555
                </li>,
              ],
              "className": "member-list",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Name:",
                },
                "ref": null,
                "rendered": "Name:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Bernie Sanders",
                },
                "ref": null,
                "rendered": "Bernie Sanders",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Party:",
                },
                "ref": null,
                "rendered": "Party:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I",
                },
                "ref": null,
                "rendered": "I",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "State:",
                },
                "ref": null,
                "rendered": "State:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "VT",
                },
                "ref": null,
                "rendered": "VT",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Official Website:",
                },
                "ref": null,
                "rendered": "Official Website:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "www.sanders.org",
                },
                "ref": null,
                "rendered": "www.sanders.org",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Twitter:",
                  "className": "show",
                },
                "ref": null,
                "rendered": "Twitter:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "sanders",
                  "className": "show",
                },
                "ref": null,
                "rendered": "sanders",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Facebook:",
                  "className": "show",
                },
                "ref": null,
                "rendered": "Facebook:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "sanders",
                  "className": "show",
                },
                "ref": null,
                "rendered": "sanders",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Phone:",
                  "className": "show",
                },
                "ref": null,
                "rendered": "Phone:",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "555-5555",
                  "className": "show",
                },
                "ref": null,
                "rendered": "555-5555",
                "type": "li",
              },
            ],
            "type": "ul",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="member"
          >
            <img
              alt="Bernie Sanders"
            />
            <ul
              className="member-list"
            >
              <li>
                Name:
              </li>
              <li>
                Bernie Sanders
              </li>
              <li>
                Party:
              </li>
              <li>
                I
              </li>
              <li>
                State:
              </li>
              <li>
                VT
              </li>
              <li>
                Official Website:
              </li>
              <li>
                www.sanders.org
              </li>
              <li
                className="show"
              >
                Twitter:
              </li>
              <li
                className="show"
              >
                sanders
              </li>
              <li
                className="show"
              >
                Facebook:
              </li>
              <li
                className="show"
              >
                sanders
              </li>
              <li
                className="show"
              >
                Phone:
              </li>
              <li
                className="show"
              >
                555-5555
              </li>
            </ul>
          </div>,
        ],
        "className": "members-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Bernie Sanders"
              />,
              <ul
                className="member-list"
              >
                <li>
                  Name:
                </li>
                <li>
                  Bernie Sanders
                </li>
                <li>
                  Party:
                </li>
                <li>
                  I
                </li>
                <li>
                  State:
                </li>
                <li>
                  VT
                </li>
                <li>
                  Official Website:
                </li>
                <li>
                  www.sanders.org
                </li>
                <li
                  className="show"
                >
                  Twitter:
                </li>
                <li
                  className="show"
                >
                  sanders
                </li>
                <li
                  className="show"
                >
                  Facebook:
                </li>
                <li
                  className="show"
                >
                  sanders
                </li>
                <li
                  className="show"
                >
                  Phone:
                </li>
                <li
                  className="show"
                >
                  555-5555
                </li>
              </ul>,
            ],
            "className": "member",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Bernie Sanders",
                "src": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    Name:
                  </li>,
                  <li>
                    Bernie Sanders
                  </li>,
                  <li>
                    Party:
                  </li>,
                  <li>
                    I
                  </li>,
                  <li>
                    State:
                  </li>,
                  <li>
                    VT
                  </li>,
                  <li>
                    Official Website:
                  </li>,
                  <li>
                    www.sanders.org
                  </li>,
                  <li
                    className="show"
                  >
                    Twitter:
                  </li>,
                  <li
                    className="show"
                  >
                    sanders
                  </li>,
                  <li
                    className="show"
                  >
                    Facebook:
                  </li>,
                  <li
                    className="show"
                  >
                    sanders
                  </li>,
                  <li
                    className="show"
                  >
                    Phone:
                  </li>,
                  <li
                    className="show"
                  >
                    555-5555
                  </li>,
                ],
                "className": "member-list",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Name:",
                  },
                  "ref": null,
                  "rendered": "Name:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Bernie Sanders",
                  },
                  "ref": null,
                  "rendered": "Bernie Sanders",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Party:",
                  },
                  "ref": null,
                  "rendered": "Party:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "I",
                  },
                  "ref": null,
                  "rendered": "I",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "State:",
                  },
                  "ref": null,
                  "rendered": "State:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "VT",
                  },
                  "ref": null,
                  "rendered": "VT",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Official Website:",
                  },
                  "ref": null,
                  "rendered": "Official Website:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "www.sanders.org",
                  },
                  "ref": null,
                  "rendered": "www.sanders.org",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Twitter:",
                    "className": "show",
                  },
                  "ref": null,
                  "rendered": "Twitter:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "sanders",
                    "className": "show",
                  },
                  "ref": null,
                  "rendered": "sanders",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Facebook:",
                    "className": "show",
                  },
                  "ref": null,
                  "rendered": "Facebook:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "sanders",
                    "className": "show",
                  },
                  "ref": null,
                  "rendered": "sanders",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Phone:",
                    "className": "show",
                  },
                  "ref": null,
                  "rendered": "Phone:",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "555-5555",
                    "className": "show",
                  },
                  "ref": null,
                  "rendered": "555-5555",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
